.golangci.yml

```diff
@@ -31,6 +31,16 @@ linters-settings:
     line-length: 170
     tab-width: 4
 
+  tagliatelle:
+    # Check the struck tag name case.
+    case:
+      rules:
+        avro: snake
+        bson: camel
+        json: snake
+        yaml: camel
+        xml: camel
```

Comment:
Right now golangci-lint broken,  @PavelKhripkov  fixed it in this PR: https://github.com/percona/pmm/pull/968
So after fix will be merged you CI will crash because we have mess with json tags, you can find both, snake case and camel case in our codebase.

Reply:
I disabled json and yaml for now. We have too many warnings popping up.
We can discuss if and how we want to address it later.